docs/api/taxons.rst

```diff
@@ -0,0 +1,858 @@
++--------------+--------------------------------------------------------------------------------------------+
+| translations | Collection of translations (each contains slug and name and description in given language) |
++--------------+--------------------------------------------------------------------------------------------+
+| position     | The position of taxon among other taxons                                                   |
++--------------+--------------------------------------------------------------------------------------------+
+| images       | Images assigned to taxon                                                                   |
++--------------+--------------------------------------------------------------------------------------------+
+| left         | Location within taxonomy                                                                   |
++--------------+--------------------------------------------------------------------------------------------+
+| right        | Location within taxonomy                                                                   |
++--------------+--------------------------------------------------------------------------------------------+
+| level        | How deep taxon is in the tree                                                              |
++--------------+--------------------------------------------------------------------------------------------+
+| children     | Sub taxons                                                                                 |
++--------------+--------------------------------------------------------------------------------------------+
+
+
+.. note::
+
+    Read more about `Taxon`__
+
+__ http://docs.sylius.org/en/latest/components/Taxonomy/models.html#taxon
+
+Collection of Taxons
+--------------------
+
+To retrieve the paginated list of taxons you will need to call the ``/api/v1/taxons/`` endpoint with ``GET`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/taxons/
+
++---------------------------------------+----------------+---------------------------------------------------+
+| Parameter                             | Parameter type | Description                                       |
++=======================================+================+===================================================+
+| Authorization                         | header         | Token received during authentication              |
++---------------------------------------+----------------+---------------------------------------------------+
+| limit                                 | query          | *(optional)* Number of items to display per page, |
+|                                       |                | by default = 10                                   |
++---------------------------------------+----------------+---------------------------------------------------+
+
+To see the first page of all taxons use the method below.
```

Comment:
⚠️  <del>the method below</del> => **the below method**, check this everywhere ; )